File: fizzbuzz.rb
def fizzbuzz(n)
  if (n % 15).zero?
    "FizzBuzz"
  elsif (n % 5).zero?
    "Buzz"
  elsif (n % 3).zero?
    "Fizz"
  else
    n
  end
end

(1..100).each {|n| puts fizzbuzz(n)}

File: fizzbuzz.clj
(defn fizzbuzz [n]
   (cond
     (zero? (mod n 15)) "FizzBuzz"
     (zero? (mod n 3)) "Fizz"
     (zero? (mod n 5)) "Buzz"
     :else n))

(run! println (map fizzbuzz (range 1 101)))